patchwork/forms.py

```diff
@@ -106,6 +107,35 @@ class DeleteBundleForm(forms.Form):
     bundle_id = forms.IntegerField(widget=forms.HiddenInput)
 
 
+class CreateNoteForm(forms.ModelForm):
+    name = 'createnoteform'
+    form_name = forms.CharField(initial=name, widget=forms.HiddenInput)
+    content = forms.CharField(label='Content', widget=forms.Textarea)
+    maintainer_only = forms.BooleanField(
+        label='Maintainers Only', initial=True, widget=forms.CheckboxInput
```

Comment:
We must set required=False for checkboxes to enable public notes creation
```suggestion
        label='Maintainers Only', initial=True, widget=forms.CheckboxInput, required=False
```